webserver/src/webserver.cpp

```diff
@@ -42,13 +43,16 @@ po::options_description commandLineArguments()
 
 int main(int argc, char* argv[])
 {
+  const std::string SERVICE_PLUGIN_DIR = "/../lib/serviceplugin";
+  const std::string WEBGUI_DIR = "/../share/codecompass/webgui/";
```

Comment:
I think we should put this below the binDir declaration:
```cpp
std::string binDir = boost::filesystem::canonical(
  boost::filesystem::path(argv[0]).parent_path()).string();

const std::string SERVICE_PLUGIN_DIR = binDir + "/../lib/serviceplugin";
const std::string WEBGUI_DIR = binDir + "/../share/codecompass/webgui/";
```
Use these variables instead of concatenation of binDir and WEBGUI_DIR/SERVICE_PLUGIN_DIR below.